src/api/entities/Venue/index.ts

```diff
@@ -122,11 +122,11 @@ export class Venue extends Entity<UniqueIdentifiers, string> {
       settlement.details(venueId),
     ]);
 
-    const { creator, venue_type: type } = venueInfo.unwrap();
+    const { creator, venueType: type } = venueInfo.unwrap();
 
     return {
       owner: new Identity({ did: identityIdToString(creator) }, context),
-      description: venueDetailsToString(details),
+      description: bytesToString(details),
```

Comment:
Somehow these lost the special type we had and are now bytes.

Would be nice to have the custom types.